### ISSUE TITLE
[Obsidian] ...Unable to Use Raycast Obsidian Extension Features with Custom Config File Name

### ISSUE THREAD

**Issue by marine7382** [raycast/extensions#8991]
### Extension

https://www.raycast.com/KevinBatdorf/obsidian

### Description

I would like to report a bug regarding the Raycast Obsidian extension. The issue arises when using a custom config file name instead of the default ﻿.obsidian.

When the Obsidian config file is set to the default name ﻿.obsidian, the Raycast Obsidian extension functions correctly and all features are accessible. However, when a custom config file name is used, the Advanced URI plugin for Obsidian fails to recognize it. As a result, the Raycast Obsidian extension displays an error message stating "Advanced URI plugin not installed," even though the plugin is actually installed and functioning properly.

### Steps To Reproduce

1. Install the Raycast Obsidian extension.
2. Set a custom name for the Obsidian config file (e.g., ﻿myconfig.obsidian).
3. Launch Raycast and activate the Obsidian extension.
4. Attempt to use any of the features provided by the Raycast Obsidian extension.

### Current Behaviour

When a custom config file name is used, the Raycast Obsidian extension fails to recognize the config file and displays the error message "Advanced URI plugin not installed" erroneously, even when the Advanced URI plugin is installed and functional.

### Expected Behaviour

The Raycast Obsidian extension should recognize the custom config file name and function properly, allowing access to all its features.

**Comment by marine7382** [comment]
Apologies if this wasn't formatted correctly. This is my first bug report. Additionally, I believe I encountered the same issue with other Raycast/Obsidian extensions. Therefore, I may be mistaken in reporting it here. If that's the case, I'm willing to close this report.

**Comment by KevinBatdorf** [comment]
Hey thanks for bringing this up. I opened a PR just now that will let you change it. But the setting is applied across all vaults. I'm not sure there is a convenient way to have a custom config filename separately per vault in the extension given how the extension is currently built. 

I may look into adding support for that, but don't want to unless someone requests it. Are you using multiple vaults with varying config names different among the vaults?

Also, just generally curious why you changed it from the default?

**Comment by marine7382** [comment]
It was just because I could. Lol But I didn't realize that changing the config file to anything like ".obsidian-main" or whatever would have an effect on the Raycast plugin until I finally realized what happened. And no, I only have one primary vault where I changed the config file name. There are a few other vaults I downloaded from GitHub, but I don't mess with those settings because they're primarily just Obsidian plugin documentation.

But I ended up changing it back to the normal ".obsidian" and it looks like it's running perfectly again. It's still one of my favorite and most used Raycast extensions! Awesome work, @KevinBatdorf !
